server/playerServer/playerserver.js

```diff
@@ -662,7 +662,7 @@ var server = http.createServer(function (req, res) {
 		var uKey = parsedUrl.query.uKey;
 		var room = parsedUrl.query.room;
 		var regionId = parsedUrl.query.regionId;
-		var overrideOwner = parsedUrl.query.overrideOwner;
+		var overrideOwner = (parsedUrl.query.overrideOwner === 'true') ? true : false; //sent in as string.
```

Comment:
all things from parsedUrl.query that isnt an array acts like a string D'; so it literally sends a string with true or false in it.